docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -511,16 +521,191 @@ To manage the media controller playlist in your application, you must handle req
+## Media Controller Server Abilities
+
+Various abilities can be set to give the information to the clients about the supported features. These abilities can be divided into two groups:
+
+1. Media controller simple abilities
+
+    Each ability is described by a single [MediaControllerAbilitySupport](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerAbilitySupport) value and is not a part of a complex ability structure:
+
+    - PLAYBACK_POSITION: Changing playback position.
+    - SHUFFLE: Changing shuffle mode.
+    - REPEAT: Changing repeat state.
+    - PLAYLIST: Adding/changing/removing playlists.
+    - CLIENT_CUSTOM: Receiving custom commands from media controller clients.
+    - SEARCH: Receiving search requests from media controller clients.
+    - SUBTITLES: Receiving requests for subtitles mode change from media controller clients.
+    - MODE_360: Receiving requests for spherical (360&deg;) mode change from media controller clients.
+
+2. Media controller complex abilities
+
+    Each ability consists of several types:
+
+    - playback: Server playback actions.
+    - displayRotation: Setting display orientations.
+    - displayMode: Setting display modes.
+
+### Setting Media Controller Server Abilities
+
+You can set the media server abilities as follows:
+
+```javascript
+mcServer.abilities.playback.next = "YES";
+mcServer.abilities.playback.prev = "YES";
+mcServer.abilities.playback.rewind = "NO";
+mcServer.abilities.playback.forward = "NO";
+
+mcServer.abilities.playback.saveAbilities();
+
+mcServer.abilities.displayMode.fullScreen = "YES";
+mcServer.abilities.displayRotation.rotation180 = "YES";
+```
+
+### Checking Media Controller Server Abilities
+
+Using `saveAbilities()` is required to save changes of playback abilities into database, otherwise changes will have no effect on the device and clients will not be notified about an update. Other abilities are updated instantly and there is no need to manually save these abilities.
+
+To get abilities of the server on the client side:
+
+```javascript
+var mcClient = tizen.mediacontroller.getClient();
+var mcServerInfo = mcClient.getLatestServerInfo();
+
+console.log("ability NEXT: " + mcServerInfo.abilities.playback.next);
+console.log("ability PREV: " + mcServerInfo.abilities.playback.prev);
+console.log("ability REWIND: " + mcServerInfo.abilities.playback.rewind);
+console.log("ability FORWARD: " + mcServerInfo.abilities.playback.forward);
+```
+
+### Monitoring Media Controller Server Abilities
+
+You can monitor changes of server abilities using the `addAbilityChangeListener` method on client side. You will also receive the notifications about every ability change on the server side:
+
+```javascript
+/* Client-side code */
+var mcClient = tizen.mediacontroller.getClient();
+
+var listener =
+{
+  onplaybackabilitychanged: function(server, ability)
+  {
+    console.log("playback ability changed, server name: " + server.name + ", abilities: ");
```

Comment:
From line 603 to line 618.

Is it ok to use server.name, not mcServer.name?

Reply:
server.name is ok for this context